Source/Helpers/Extensions/UINavigationBar+Extensions.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UINavigationBar+Extensions.swift
+//  YPImagePicker
+//
+//  Created by Sebastiaan Seegers on 02/03/2020.
+//  Copyright © 2020 Yummypets. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+
+extension UINavigationBar {
+
+    func setTitleFont(font: UIFont?) {
+        guard font != nil else { return }
```

Comment:
Could be better if you unwrap here, so that you can avoid force unwrapping at below line.
```suggestion
        guard let font = font  else { return }
```